### PR TITLE
[ioctl] add did service command

### PR DESCRIPTION
**Pull request by ququzone**
# Description
Add ioctl command for DID service

Fixes #(issue)

## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Code refactor or improvement
- [] Breaking change (fix or feature that would cause a new or changed behavior of existing functionality)
- [] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] make test
- [] fullsync
- [] Other test (please specify)

**Test Configuration**:
- Firmware version:
- Hardware:
- Toolchain:
- SDK:

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by CoderZhi — APPROVED** [review]

File: ioctl/cmd/did/didserviceadd.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package did
+
+import (
+	"encoding/json"
+
+	"github.com/iotexproject/iotex-core/ioctl/cmd/action"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/spf13/cobra"
```
Comment (dustinxie) [inline ioctl/cmd/did/didserviceadd.go:9]: move to line 5

File: ioctl/cmd/did/didserviceadd.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		Signature:       *signature,
+		Tag:             args[1],
+		Type:            args[2],
+		ServiceEndpoint: args[3],
+	}
+	serviceBytes, err := json.Marshal(&serviceReq)
+	if err != nil {
+		return output.NewError(output.ConvertError, "failed to encode request", err)
+	}
+
```
Comment (dustinxie) [inline ioctl/cmd/did/didserviceadd.go:58]: remove empty line

File: ioctl/cmd/did/didserviceremove.go [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+	serviceReq := &ServiceRemoveRequest{
+		Signature: *signature,
+		Tag:       args[1],
+	}
+	serviceBytes, err := json.Marshal(&serviceReq)
+	if err != nil {
+		return output.NewError(output.ConvertError, "failed to encode request", err)
+	}
+
```
Comment (dustinxie) [inline ioctl/cmd/did/didserviceremove.go:56]: remove empty line

**Review by dustinxie — APPROVED** [review]

File: ioctl/cmd/did/didserviceadd.go (hunk 1 above)
Comment (ququzone) [reply ioctl/cmd/did/didserviceadd.go]: done

**Pull request merged**